suplemon/modules/system_clipboard.py

```diff
@@ -12,8 +12,10 @@ def init(self):
             self.clipboard_type = "xsel"
         elif self.has_pb_support():
             self.clipboard_type = "pb"
+        elif self.has_xclip_support():
+            self.clipboard_type = "xclip"
         else:
-            self.logger.warning("Can't use system clipboard. Install 'xsel' or 'pbcopy' for system clipboard support.")
+            self.logger.warning("Can't use system clipboard. Install 'xsel' or 'pbcopy' or 'xclip' for system clipboard support.")
```

Comment:
This edit makes the line exceed 120 characters, causing lint to fail. You should be able to resolve that by breaking this up into multiple lines:

```python
self.logger.warning("Can't use system clipboard. " + 
                    "Install 'xsel', 'pbcopy', or 'xclip' for system clipboard support.")
```

Reply:
The fix is totally fine, but it makes me wonder out of curiosity whether this is the only way to break strings up in python if line length is a limit? Using `"""` would obviously introduce newlines and extra whitespace.

Reply:
I guess breaking the string apart and chaining them with `+` is the first thing that comes to mind when the line is very long, like much longer than the one in question, but then you'll have to remember putting spaces between two substrings. Or maybe something like `' '.join([...])`?